src/itdelatrisu/opsu/translation/en_US.lang

```diff
@@ -0,0 +1,371 @@
+ui.startup.skins=Unpacking new skins...
+ui.startup.replays=Importing replays...
+ui.startup.sounds=Loading sounds...
+
+# Updater status messages
+
+notifications.update.available=A new update is available!
+notifications.update.download.failed=Update failed due to a connection error.
+notifications.update.download.complete=Update has finished downloading.
+notifications.update.completed=opsu! is now up to date!
+notifications.update.completed.view=opsu! is now up to date!\nClick here to see what's changed!
+
+ui.updater.state.checking=Checking for updates...
+ui.updater.state.error.connection=Connection error
+ui.updater.state.error.internal=Internal error
+ui.updater.state.latest=Up to date!
+ui.updater.state.update.available=Update available!\nClick to download.
+ui.updater.state.update.downloaded=Download complete.\nClick to restart.
+ui.updater.state.update.downloading=Downloading update...
+ui.updater.state.update.downloading.progress=Downloading update...\n%.1f%% complete (%s/%s)
+ui.updater.state.update.queued=Update queued.
+
+# ErrorHandler messages
+errorhandler.screen.title=Error
+errorhandler.screen.desc=An error occurred. :(
+errorhandler.screen.desc.report=Something bad happened. Please report this!
+errorhandler.log.view=View Error Log
+errorhandler.log.send=Send Report
+
+# Options
+options.name=Options
+options.desc=Change the way opsu! behaves
+options.search=Type to search!
+options.value.disabled=Disabled
+
+options.graphics.fullscreen=Fullscreen mode
+options.graphics.fullscreen.desc=Switch to dedicated fullscreen mode
+options.graphics.resolution=Resolution
+options.graphics.resolution.desc=Change the in-game resolution
+options.graphics.framesync=Frame limiter
+options.graphics.framesync.desc=Higher values means higher CPU usage
+options.graphics.showFPS=Show FPS counter
+options.graphics.showFPS.desc=Show a subtle FPS counter in the bottom right corner of the screen
+options.graphics.screenshotFormat=Screenshot format
+options.graphics.screenshotFormat.desc=Press F12 to take a screenshot.
+
+options.graphics.sliders.snaking=Snaking sliders
+options.graphics.sliders.snaking.desc=Sliders gradually snake out from their starting point.
+options.graphics.sliders.beta=Use experimental sliders
+options.graphics.sliders.beta.desc=Render sliders using the experimental slider style.
+options.graphics.sliders.beta.caps=Draw slider caps
+options.graphics.sliders.beta.caps.desc=Draw caps (end circles) on sliders.\nOnly applies to experimental sliders.
+options.graphics.sliders.beta.shrink=Shrinking sliders
+options.graphics.sliders.beta.shrink.desc=Sliders shrink toward their ending point when the ball passes.\nOnly applies to experimental sliders.
+options.graphics.sliders.beta.merge=Merging sliders
+options.graphics.sliders.beta.merge.desc=For overlapping sliders, don't draw the edges and combine the slider tracks where they cross.\nOnly applies to experimental sliders.
+
+options.language=Select language:
```

Comment:
For consistency with the other options, I would make this just "Language", or possibly "Interface language" or "UI language".